src/test/java/com/openlattice/chronicle/ChronicleServerTests.java

```diff
@@ -1,11 +1,497 @@
+
+    // test app data
+    private static final Pair<String, String>         CAMERA                    = Pair
+            .of( "com.android.camera2", "Camera" );
+    private static final Pair<String, String>         GMAIL                     = Pair
+            .of( "Gmail", "com.google.android.gm" );
+    private static final Pair<String, String>         YOUTUBE                   = Pair
+            .of( "come.google.android.youtube", "YouTube" );
+    private static final Pair<String, String>         CHROME                    = Pair
+            .of( "com.android.chrome", "Chrome" );
+    private static final Pair<String, String>         MAPS                      = Pair
+            .of( "com.google.android.apps.maps", "Maps" );
+    private static final Pair<String, String>         PHONE                     = Pair
+            .of( "com.android.dialer", "Phone" );
+
+    private static       Map<String, UUID>            entitySetNameIdMap        = new HashMap<>();
+    private static       UUID                         fullNamePTID;
+    private static       UUID                         durationPTID;
+    private static       UUID                         titlePTID;
+    private static       UUID                         startDateTimePTID;
+    private static       UUID                         dateLoggedPTID;
+    private static UUID recordTypePTID;
+
+    private static DataApi       dataApi;
+    private static EdmApi        edmApi;
+    private static SearchApi     searchApi;
+    private static EntitySetsApi entitySetsApi;
+
+    ChronicleConfiguration configuration = new ChronicleConfiguration( "user", "password" );
+    @Inject
+    private EventBus eventBus;
+    ChronicleService chronicleService = new ChronicleServiceImpl( eventBus, configuration );
+
+    public ChronicleServerTests() throws ExecutionException {
+
+    }
+
+    @BeforeClass
+    public static void chronicleServerTest() throws Exception {
+
+        // replace this with an actual auth0_id token
+        String token = "";
+
+        LoadingCache<Class<?>, ApiClient> apiClientCache = CacheBuilder
+                .newBuilder()
+                .expireAfterWrite( Duration.ofHours( 10 ) )
+                .build( new CacheLoader<Class<?>, ApiClient>() {
+                    @Override
+                    public ApiClient load( Class<?> key ) throws Exception {
+                        // String token = MissionControl.getIdToken( TEST_USER, TEST_PASSWORD );
+                        return new ApiClient( RetrofitFactory.Environment.TESTING, () -> token );
+                    }
+                } );
+        ApiClient apiClient = apiClientCache.get( ApiClient.class );
+
+        dataApi = apiClient.getDataApi();
+        edmApi = apiClient.getEdmApi();
+        entitySetsApi = apiClient.getEntitySetsApi();
+        searchApi = apiClient.getSearchApi();
+
+        setUpTestingEnvironment();
+    }
```

Comment:
I have a feeling this is not quite right. Let's get others to look at the setup.

Reply:
Dropping tests for now.